src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,28 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class WinningNumbers {
+	private final Set<Integer> winningNumbers;
+
+	public WinningNumbers(Set<Integer> winningNumbers) {
+		if(winningNumbers.size() != 6) {
```

Comment:
`Set`을 활용한 중복제거 좋습니다 👍 
컨벤션 한번 신경써 주시고
`6`은 매직넘버이니 추출되면 좋을 것 같습니다 🙇 
> https://hoonmaro.tistory.com/44
```suggestion
		if (winningNumbers.size() != 6) {
```